applications/launchpad/gui-react/src/store/mining/selectors.ts

```diff
@@ -53,6 +53,11 @@ export const selectTariSetupRequired = createSelector(
 export const selectMergedMiningState = (r: RootState) => r.mining.merged
 export const selectMergedMiningAddress = (r: RootState) =>
   r.mining.merged.address
+export const selectMoneroUrls = (r: RootState) => {
```

Comment:
you can use map to avoid clunky pushing
`=> (r.mining.merged.urls || []).map(u => u.url).join(',')`